src/translations/translations.json

```diff
@@ -553,6 +557,8 @@
       "Status": "Status",
       "Server": "Serveur",
       "Owner": "Propriétaire",
+      "ConnectButton": "Se connecter",
```

Comment:
You can just DeepL in order to get the JP translations as well. Make sure it's included as well!

Reply:
English / French OK but i don't speak JP and can't maintain all the languages :/